ui/resources/docker/Rockerfile

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2015 EMC Corporation
+# All Rights Reserved
+#
+# This software contains the intellectual property of EMC Corporation
+# or is licensed to EMC Corporation from third parties.  Use of this
+# software and the intellectual property contained therein is expressly
+# limited to the terms and conditions of the License Agreement under which
+# it is provided by or on behalf of EMC.
+
+FROM {{ .FromImage }}
+MAINTAINER Travis Wichert <travis.wichert@emc.com>
+ENV ANSIBLE_CONFIG="/etc/ansible/ansible.cfg"
+ENV ANSIBLE_HOSTS="/usr/local/src/ui/inventory.py"
+
+COPY ui/resources/docker/ecs-install-requirements.txt /etc/ecs-install-requirements.txt
+COPY ui/resources/docker/apk-repositories /etc/apk/repositories
+
+ENV http_proxy={{ .HTTPProxy }}
```

Comment:
These two lines make rocker fail when building the image and no HTTPProxy and PipProxy is used. It can be fixed by checking if these vars are set.

```
{{ if .HTTPProxy }}
ENV http_proxy={{ .HTTPProxy }}
{{ end }}

{{ if .PipProxy }}
ENV pip_proxy={{ .PipProxy }}
{{ end }}
```